public/data/sachinparihar.json

```
@@ -0,0 +1,22 @@
+{
+    "name": "Sachin Parihar",
+    "type": "personal",
+    "bio": "I am Sachin Parihar.Currently, I am pursuing my graduation in Computer Science from GJU College.I am a Open Source Enthusiast! & Also, interested in DevOps!",
+    "avatar": "https://github.com/sachinparihar.png",
+    "links": [{
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/sachinparihar",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/@SachinSarkar001",
+            "icon": "twitter"
+        },
+        {
+            "name": "Follow me on Instagram",
+            "url": "https://www.instagram.com/sachin_parihar001/",
+            "icon": "instagram"
+        }
+    ]
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```